themes/default/content/templates/static-website/_index.md

```diff
@@ -1,9 +1,33 @@
 ---
 title: Static Website Templates
 layout: overview
+description: Pulumi program templates are the fastest way to deploy static websites on [AWS](/aws), [Azure](/azure), or [Google Cloud Platform](/gcp). Templates come with predefined [infrastructure as code](/what-is/what-is-infrastructure-as-code) so you can get started instantly.
 meta_desc: Pulumi program templates that make it easy to deploy static websites on AWS, Azure, or Google Cloud Platform.
 meta_image: meta.png
 weight: 1
 ---
 
-Pulumi program templates that make it easy to deploy static websites on AWS, Azure, or Google Cloud Platform.
+
+### What are static websites?
+
+Static websites are an efficient and scalable way of building and serving websites created with HTML, CSS, or JavaScript. Popular frameworks for static websites include Next.js, Hugo, Gatsby, Jekyll, and more.
+
+**On AWS,** you can host a static website by configuring Amazon S3 for hosting and Amazon CloudFront as the CDN. You could also use AWS Amplify, which is designed for deploying static websites more easily.
+
+**On Azure,** you can host a static website by configuring Azure Blob Storage for hosting and Azure CDN for content delivery. You could also use Azure Static Web Apps, which provides a CI/CD pipeline for deploying static websites.
+
+**On Google Cloud Platform,** you can host a static website by configuring Cloud Storage for hosting and Cloud Load Balancing with HTTPS configured.
+
+### Building and deploying static websites on AWS, Azure, and Google Cloud
+
+Infrastructure as code is an efficient and repeatable way of building a static website with programming languages and deploying the website to your preferred cloud.
+
+Pulumi’s open source, infrastructure as code SDK that lets you build and deploy static websites with TypeScript/JavaScript, Python, Go, Java, .NET, and YAML. The main benefits include:
+
+**Programming Languages:** Define infrastructure as code in your favorite language instead of domain-specific languages or clicking through cloud consoles.
+
+**Easier Integration:** Integrate every cloud service needed to run your static website into a single program.
+
+**Fast, Easy Deployment:** Quickly deploy your static website with a CLI or from a CI/CD workflow.
+
+**Rapid Development:** Author, version, test, and release infrastructure changes just like software.
```

Comment:
```suggestion
* **Programming Languages:** Define infrastructure as code in your favorite language instead of domain-specific languages or clicking through cloud consoles.

* **Easier Integration:** Integrate every cloud service needed to run your static website into a single program.

* **Fast, Easy Deployment:** Quickly deploy your static website with a CLI or from a CI/CD workflow.

* **Rapid Development:** Author, version, test, and release infrastructure changes just like software.
```